wallet/file-format.md

```diff
@@ -50,6 +50,9 @@ Specifically, the PBKDF2 arguments defined in section 5.2 of the aforementioned
 - 600,000 iterations, and a
 - derived key length of 32 bytes.
 
+The salt should respect the recommended practices in [PHC string format specification](https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md#function-duties), which states the following:
+> The role of salts is to achieve uniqueness. A random salt is fine for that as long as its length is sufficient; a 16-byte salt would work well (by definition, UUID are very good salts, and they encode over exactly 16 bytes). 16 bytes encode as 22 characters in B64.
+
```

Comment:
I think it doesn't bring value to copy-paste the sentence from that specification. Also we already cited https://www.ietf.org/rfc/rfc2898.txt that says:

> then the salt may be generated at random and
         need not be checked for a particular format by the party
         receiving the salt. It should be at least eight octets (64
         bits) long.

That's why I think that it's not necessary to cite PHC string format specification.

In addition, I think it's more in the Serialization of the account file that we should talk about string format: base 64 encoding; because this section is dedicated to cryptography.

In the Serialization bellow, we have to change the change the examples and replace "Salt for PBKDF2 (16 Bytes)" by something like "Salt for PBKDF2 (16 Bytes, encoded in base 64)"

Reply:
I have no issues with your suggestions, as you were part of this spec redaction I will let you apply them to avoid more friction.